src/server.rs

```diff
@@ -251,6 +252,31 @@ impl RelayerApi {
             ));
         }
 
+        let relayer_queued_tx_gas_limit_sum = app
+            .db
+            .get_relayer_pending_txs_gas_limit_sum(api_token.relayer_id())
+            .await?;
+
+        let block_fees = app.db.get_latest_block_fees_by_chain_id(relayer.chain_id).await?;
+        if let Some(block_fees) = block_fees {
```

Comment:
I decided to check funds only if estimates are available. Other option is to return error here if we think it is crucial for us.